pkg/client/client.go

```diff
@@ -297,3 +297,16 @@ func DefaultOpenShiftUserAgent() string {
 	version = seg[0]
 	return fmt.Sprintf("%s/%s (%s/%s) openshift/%s", path.Base(os.Args[0]), version, runtime.GOOS, runtime.GOARCH, commit)
 }
+
+type HybridClient struct {
```

Comment:
See how the Kubernetes client is currently implemented: https://github.com/kubernetes/kubernetes/blob/071d21257fdfd439a9286fffbd5972c85643cd49/pkg/client/unversioned/client.go#L128-l134

We should do the same for our client